iac-extensions/jvm-framework-config/src/main/java/org/sonar/iac/jvmframeworkconfig/checks/springconfig/ExcessiveFileUploadSizeLimitCheck.java

```diff
@@ -17,12 +17,14 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.iac.jvmframeworkconfig.checks;
+package org.sonar.iac.jvmframeworkconfig.checks.springconfig;
 
 import java.util.Locale;
 import java.util.Set;
 import java.util.regex.Pattern;
+
 import javax.annotation.CheckForNull;
+
```

Comment:
It looks like you have the wrong settings for imports, there should not be a line break in between 
https://github.com/SonarSource/sonar-developer-toolset

Reply:
Ahh, sorry it happened again.
I have good settings, but dunno why my local IaC project doesn't want to accept them.
I had to clone the repo again to finally make it work, but I mistakenly worked on the previous clone.

Reply:
I will create a ticket to add import formatting to spotless, we shouldn't rely on our individual machine here.